front50-sql/src/main/resources/db/changelog/20191112-modify-char-columns-postgres.yml

```diff
@@ -0,0 +1,89 @@
+databaseChangeLog:
```

Comment:
This is a new changeset only applied for Postgres.  CHAR types are converted to bpchar (blank pad characters) in Postgres and they are not recommended datatypes + JOOQ doesn't play well with padded column values. So this change set modifies the column data type to VARCHAR.

Reply:
I did a quick look around but came up short - could you link where it's documented that `char` / `bpchar` is not recommended? I'd like to get more learned on this.

Reply:
https://stackoverflow.com/questions/51528982/what-comparison-problems-can-using-the-varchar-type-instead-of-the-char-type-avo